cli/cmd_report.cpp

```diff
@@ -228,6 +228,10 @@ class report_console_hooks : public drivers::scan_results::base_hooks {
     }
 
     /// Counts how many results of a given type have been received.
+    ///
+    /// \param type Test result type to count results for.
+    ///
+    /// \return The number of test results with `type`.
```

Comment:
I don't think we use the backticks style anywhere. Same thing throughout the change.

Reply:
Ok. I tried using markdown for clarity (which one can do with Doxygen).

Reply:
Oh I didn't know we could use Markdown syntax. OK.